django_irods/utils.py

```diff
@@ -50,3 +50,16 @@ def normalized_bucket_name(username):
         row = cursor.fetchone()
         bucket_name = row[0]
         return bucket_name
+
+
+def is_metadata_xml_file(file_path):
+    """Determine whether a given file is metadata.
+    Note: this will return true for any file that ends with the metadata endings
+    We are taking the risk that user might create a file with the same filename ending
```

Comment:
We could probably remove the risk if also check if this file is not in Django db.